docs/releasing.md

```diff
@@ -0,0 +1,138 @@
+# Releasing ironic-image
+
+This document explains how to prepare and publish a release for ironic-image, and
+how often a new release branch is created.
```

Comment:
The "How often" part could use move love. The way I read the document, we should aim for a release shortly after a corresponding Ironic stable or bugfix branch is created, otherwise the newly created ironic-image branch will effectively go back in time once you pin the projects.

Reply:
I've added some words in the Branches lifecycle paragraph